main.go

```diff
@@ -130,7 +130,8 @@ const (
 )
 
 const (
-	commandStartPipeline = "start pipeline"
+	commandStartPipeline    = "start pipeline"
+	commandCherryPickBranch = "cherry pick to:"
```

Comment:
I personally always type "cherry-pick". Could it be possible to make the bot accept both space and `-`? Otherwise is just fine, I only need to get used to.

Reply:
Agreed, `cherry-pick` it is ;)